components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -697,9 +708,10 @@ public void setAccessTokenState(String accessToken, String tokenState, String to
 	    try {
 		    connection = JDBCPersistenceManager.getInstance().getDBConnection();
 	    } catch (IdentityException e) {
-		    throw new IdentityOAuth2Exception("Error occurred while trying to get a Identity " +
-		                                      "persistence store instance");
-	    }
+            throw new IdentityOAuth2Exception("Error occurred while trying to get a Identity persistence store " +
+                    "instance", e);
+
```

Comment:
Fixed with c1aca501aedd8f5adf4cc70d6c9580a8cc73fbd0 and 2a9c88f26c5cfa0c2e38db061482600fe9b218a0